tests/test.py

```diff
@@ -21,6 +22,24 @@
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
 from binhosimulators import BinhoSupernovaSimulator
 
+counter = 0
+last_ibi = Event()
+caught_ibis = []
+
+def _is_ibi(name, message):
+        return message['name'].strip() == "I3C IBI NOTIFICATION" and message['header']['type'] == "IBI_NORMAL"
+
+def _handle_ibi(name, message):
+    global counter
+    global last_ibi
+    global caught_ibis
+
+    caught_ibis.append({'dynamic_address': message['header']['address'],  'controller_response': message['header']['response'], 'mdb':message['payload'][0]})
```

Comment:
PEP8 doesn't recommend to use one type of quotes over the other (single vs double). However, it recommends to stick to one type and avoid having both types in the code. Since we are mostly using double quotes, I would suggest to change them here to double.